tests/test_appengine_adapter.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+"""Tests for the AppEngineAdapter."""
+import sys
+
+import mock
+import pytest
+import requests
+
+if requests.__build__ >= 0x021000:
+    from requests.packages.urllib3.contrib import appengine as urllib3_appengine
+    from requests_toolbelt.adapters import appengine
+else:
+    appengine = urllib3_appengine = None
+
+
+@pytest.mark.skipif(sys.version_info >= (3,),
+                    reason="App Engine doesn't support Python 3 (yet)")
```

Comment:
Also, why are we skipping Python 3 tests? Do you mean App Engine *the service* doesn't support Python 3? If so that sucks for its users, but we should still be testing on Python 3.